.azdo/pipelines/azure-pipelines.yml

```
@@ -177,7 +177,7 @@ stages:
         DeployRedis: ${{ parameters.DeployRedis }}
 
   - stage: vnexte2eDeploy
-    dependsOn:
+    dependsOn: 
```

Please remove whitespace added to end of line. No point in having more changed files than needed in PR! Thanks.
```suggestion
    dependsOn:
```